pkg/vendir/fetch/image/imgpkg_test.go

```diff
@@ -239,20 +239,20 @@ func requireImgpkgEnv(t *testing.T, expectedEnv, actualEnv []string) {
 	require.Equal(t, expectedEnv, filteredActualEnv)
 }
 
-type dummyCache struct {
+type sampleCache struct {
```

Comment:
After some soul searching this in fact looks much more like a fake [test double](https://en.wikipedia.org/wiki/Test_double) than really a dummy object test double.
```suggestion
type fakeCache struct {
```